src/bnb_trading/backtester.py

```
@@ -105,29 +105,31 @@
 from tqdm import tqdm
 
 # Add current directory to Python path for module imports
 sys.path.append(os.path.dirname(os.path.abspath(__file__)))
 
-from data_fetcher import BNBDataFetcher
-from fibonacci import FibonacciAnalyzer
-from weekly_tails import WeeklyTailsAnalyzer
-from indicators import TechnicalIndicators
-from signal_generator import SignalGenerator
+# Import after path setup to avoid import errors
+from data_fetcher import BNBDataFetcher  # noqa: E402
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)